### PR TITLE
Support setting any attribute in Spec (through CR Spec Validation)

### PR DESCRIPTION
**Pull request by sanatnayar**
Created a new `validateCR` function that increases modularity (ability to add new spec attributes in the future without needing to refactor) and adds support for `qliksense config set <any_attribute>=<any_value>`. Also adding to spec is now case in-sensitive to improve user experience.

### CODE REVIEW

File: pkg/qliksense/operator.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ func (q *Qliksense) ViewOperatorController() error {
 func (q *Qliksense) GetOperatorCRDString() string {
 	result := ""
 	for _, v := range q.getFileList("crd") {
+		fmt.Println(v)
```
Comment (ffoysal) [inline pkg/qliksense/operator.go]: is needed?

File: pkg/qliksense/operator.go [hunk 2]
```diff
@@ -38,6 +39,7 @@ func (q *Qliksense) GetOperatorControllerString() string {
 
 func (q *Qliksense) getYamlFromPackrFile(packrFile string) string {
 	s, err := q.CrdBox.FindString(packrFile)
+	fmt.Println(q.CrdBox.Path)
```
Comment (ffoysal) [inline pkg/qliksense/operator.go]: is it needed

**Comment by ffoysal** [comment]
can we have various test scenario for validateCR function

**Comment by sanatnayar** [comment]
> can we have various test scenario for validateCR function

There are some tests already being performed that use this function in `context_configs_test.go` under `TestSetOtherConfigs` however I will add additional tests that test previously unsupported `set` commands for git

**Pull request merged**